test/rng_testsuite/statistics_tests/bernoulli_distribution_test.pass.cpp

```diff
@@ -0,0 +1,429 @@
+// -*- C++ -*-
+//===-- bernoulli_distribution_test.cpp ------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+//
+// Abstract:
+//
+// Test of bernoulli_distribution - check statistical properties of the distribution
+
+#include "support/utils.h"
+#include <iostream>
+
+#if TEST_DPCPP_BACKEND_PRESENT && TEST_UNNAMED_LAMBDAS
+#    include <CL/sycl.hpp>

```

Comment:
please, make it `#include` not `#    include`

Reply:
done